src/compiler/docs/generate-doc-data.ts

```diff
@@ -50,11 +82,12 @@ const getDocsComponents = async (
       const usagesDir = normalizePath(join(dirPath, 'usage'));
       const readme = await getUserReadmeContent(compilerCtx, readmePath);
       const usage = await generateUsages(compilerCtx, usagesDir);
+
       return moduleFile.cmps
         .filter((cmp: d.ComponentCompilerMeta) => !cmp.internal && !cmp.isCollectionDependency)
         .map((cmp: d.ComponentCompilerMeta) => ({
           dirPath,
-          filePath: relative(config.rootDir, filePath),
+          filePath: normalizePath(relative(config.rootDir, filePath), false),
```

Comment:
so this change was just to ensure that the paths in here would look like `foo/bar.ts` regardless of platform - we would have had that previously because of the patched `path` methods, but this additionally normalization is needed here for it now